rest-api/alembic/versions/29e344f31b21_drop_qa_columns.py

```diff
@@ -0,0 +1,263 @@
+          qra.question_id = qq.questionnaire_question_id AND
+          qq.code_id = c.code_id AND c.value = 'Race_WhatRaceEthnicity' AND
+          qr.participant_id = p.participant_id AND
+          qra.value_code_id = ac.code_id AND
+          ac.value != 'WhatRaceEthnicity_Hispanic'
+   ) race_codes,
+   (SELECT COUNT(ac.value)
+    FROM questionnaire_response qr, questionnaire_response_answer qra,
+      questionnaire_question qq, code c, code ac
+    WHERE qra.end_time IS NULL AND
+      qr.questionnaire_response_id = qra.questionnaire_response_id AND
+      qra.question_id = qq.questionnaire_question_id AND
+      qq.code_id = c.code_id AND c.value = 'Race_WhatRaceEthnicity' AND
+      qr.participant_id = p.participant_id AND
+      qra.value_code_id = ac.code_id AND
+      ac.value = 'WhatRaceEthnicity_Hispanic'
+   ) hispanic
+ FROM
+   participant p
+     LEFT OUTER JOIN hpo ON p.hpo_id = hpo.hpo_id
+     LEFT OUTER JOIN participant_summary ps ON p.participant_id = ps.participant_id
+     LEFT OUTER JOIN code state_code ON ps.state_id = state_code.code_id
+     LEFT OUTER JOIN code recontact_method_code ON ps.recontact_method_id = recontact_method_code.code_id
+     LEFT OUTER JOIN code language_code ON ps.language_id = language_code.code_id
+     LEFT OUTER JOIN code gender_code ON ps.gender_identity_id = gender_code.code_id
+     LEFT OUTER JOIN code sex_code ON ps.sex_id = sex_code.code_id
+     LEFT OUTER JOIN code sexual_orientation_code ON ps.sexual_orientation_id = sexual_orientation_code.code_id
+     LEFT OUTER JOIN code education_code ON ps.education_id = education_code.code_id
+     LEFT OUTER JOIN code income_code ON ps.income_id = income_code.code_id
+     WHERE (ps.email IS NULL OR ps.email NOT LIKE '%@example.com') AND
+           (hpo.name IS NULL OR hpo.name != 'TEST')
+"""
+
+_QUESTIONNAIRE_RESPONSE_ANSWER_VIEW_SQL = """
+CREATE OR REPLACE VIEW questionnaire_response_answer_view AS
+ SELECT
+   p.participant_id participant_id,
+   YEAR(p.sign_up_time) participant_sign_up_year,
+   p.withdrawal_status participant_withdrawal_status,
+   YEAR(p.withdrawal_time) participant_withdrawal_year,
+   p.suspension_status participant_suspension_status,
+   YEAR(p.suspension_time) participant_suspension_year,
+   hpo.name hpo,
+   (SELECT GROUP_CONCAT(concept_code.value SEPARATOR ',') FROM questionnaire_concept qc
+     INNER JOIN code concept_code ON concept_code.code_id = qc.code_id
+     WHERE qc.questionnaire_id = q.questionnaire_id) module,
+   q.questionnaire_id questionnaire_id,
+   qr.questionnaire_response_id questionnaire_response_id,
+   YEAR(qr.created) questionnaire_response_submission_year,
+   qc.value question_code,
+   qc.code_id question_code_id,
+   YEAR(qra.end_time) answer_end_year,
+   ac.value answer_code,
+   ac.code_id answer_code_id,
+   qra.value_boolean answer_boolean,
+   IF(qra.value_decimal IS NULL, 0, 1) answer_decimal_present,
```

Comment:
This seems fine. If at some point we do have decimal answers we'll need to consider having a whitelist for them. My guess is they wouldn't be PII, but better safe than sorry I guess.